scripts/openai_coverage.py

```
@@ -0,0 +1,259 @@
+    if extra_flags:
+        cmd.extend(extra_flags)
+    result = subprocess.run(cmd, capture_output=True, text=True)
+    if result.returncode != 0 and not result.stdout:
+        raise RuntimeError(f"oasdiff {subcommand} failed: {result.stderr}")
+    return json.loads(result.stdout) if result.stdout else {}
+
+
+def _check_breaking(spec: str) -> bool:
+    if not Path(spec).exists():
+        return True
+    result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)